src/structures/Presence.js

```diff
@@ -150,15 +151,12 @@ class Activity {
     this.assets = data.assets ? new RichPresenceAssets(this, data.assets) : null;
 
     this.syncID = data.sync_id;
-    this._flags = data.flags;
-  }
 
-  get flags() {
-    const flags = [];
-    for (const [name, flag] of Object.entries(ActivityFlags)) {
-      if ((this._flags & flag) === flag) flags.push(name);
-    }
-    return flags;
+    /**
+     * Flags that describe the activity
+     * @type {ActivityFlags}
+     */
+    this.flags = new ActivityFlags(data.flags);
```

Comment:
`data.flags` is undefined if no flags were set and causing an error being thrown as undefined is not a valid flag.
Either nulling the field or defaulting flags to 0 would be an option solve this.